test/topology/test_topology_remove_decom.py

```diff
@@ -135,3 +135,18 @@ async def test_rebuild_node(manager: ManagerClient, random_tables: RandomTables)
     servers = await manager.running_servers()
     await manager.rebuild_node(servers[0].server_id)
     await check_token_ring_and_group0_consistency(manager)
+
+@pytest.mark.asyncio
+async def test_concurrent_removenode(manager: ManagerClient):
+    [s1, s2, s3] = await manager.servers_add(3)
+
+    await manager.server_stop_gracefully(s3.server_id)
+
+    try:
+        await asyncio.gather(*[manager.remove_node(s1.server_id, s3.server_id), 
+                manager.remove_node(s2.server_id, s3.server_id)])
+    except Exception as e: 
+        print(e)
+        logger.info("exception raised due to concurrent remove node requests")
```

Comment:
Why not use `logger.info(f"exception raised due to concurrent remove node requests: {e}")`? Although it's subjective I guess.

Reply:
Done